CMakeLists.txt

```diff
@@ -19,9 +19,12 @@ project(OpenFAST CXX C Fortran)
 
 include(${CMAKE_SOURCE_DIR}/cmake/OpenfastCmakeUtils.cmake)
 include(${CMAKE_SOURCE_DIR}/cmake/OpenfastFortranOptions.cmake)
+include (FindPackageHandleStandardArgs)
 
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
 
+set(CMAKE_Fortran_MODULE_DIRECTORY ${CMAKE_SOURCE_DIR}/build/ftnmods)
```

Comment:
I also realised my comment was wrong when I went to examine this. I'm not yet sure what the issue really is, but I think it might be something to do with the fact that the cmake files now separately specify the target libraries and object libraries. Possibly it need to be changed so the target 'real' libraries all use the object libraries, so cmake doesn't attmept to build the same object twice.